pkg/generator/plugin/modelgen/modelgen.go

```diff
@@ -200,17 +200,19 @@ func getOverrideDataType(
 	columnName string,
 	overrides map[string]interface{},
 ) (string, bool) {
-
-	models := overrides[OverrideModelsConfig]
-	if models != nil {
-		if fieldsMap, ok := models.(map[string]interface{})[tableName]; ok {
-			fields := fieldsMap.(map[string]interface{})[OverrideFieldsConfig]
-			if _, ok := fields.(map[string]interface{})[columnName]; ok {
-				overrideTypeMap := fields.(map[string]interface{})[columnName]
-				overrideType := overrideTypeMap.(map[string]interface{})[OverrideTypeConfig]
-				return fmt.Sprintf("%v", overrideType), true
+	if overrides != nil {
+		models := overrides[OverrideModelsConfig]
+		if models != nil {
+			if fieldsMap, ok := models.(map[string]interface{})[tableName]; ok {
+				fields := fieldsMap.(map[string]interface{})[OverrideFieldsConfig]
+				if _, ok := fields.(map[string]interface{})[columnName]; ok {
+					overrideTypeMap := fields.(map[string]interface{})[columnName]
+					overrideType := overrideTypeMap.(map[string]interface{})[OverrideTypeConfig]
+					return fmt.Sprintf("%v", overrideType), true
+				}
 			}
 		}
 	}
+
```

Comment:
Might be a little cleaner to check this first instead of nesting another layer of checks.
```
if len(overrides) == 0 {
return "", false
}
```
or 
```
if overrides == nil {
return "", false
}
```

Reply:
gotcha